OpenDreamClient/OpenDream.cs

```diff
@@ -108,6 +111,8 @@ class OpenDream : Application {
                 Console.WriteLine("Connection was unsuccessful: " + pConnectionResult.ErrorMessage);
                 DisconnectFromServer();
             }
+            PacketClientData pClientData = new(new ClientData());
+            Connection.SendPacket(pClientData);
```

Comment:
This ends up sending the client data after the server approves/disapproves a connection. DM code runs before this happens (for example, `/client.New()`), and might try to check the client's timezone before it's sent.

Including the `ClientData` object in `PacketRequestConnect` might be best.

Reply:
Wasn't sure if that was OK - will do.